addons/l10n_kz/data/account_account_template_data.xml

```diff
@@ -0,0 +1,2017 @@
+        <field name="name">Краткосрочные финансовые активы</field>
+        <field name="code">1100</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1200" model="account.account.template">
+        <field name="name">Краткосрочная дебиторская задолженность</field>
+        <field name="code">1200</field>
+        <field name="reconcile" eval="True"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_payable"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1300" model="account.account.template">
+        <field name="name">Запасы</field>
+        <field name="code">1300</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1400" model="account.account.template">
+        <field name="name">Текущие налоговые активы</field>
+        <field name="code">1400</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1500" model="account.account.template">
+        <field name="name">Долгосрочные активы, предназначенные для продажи</field>
+        <field name="code">1500</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1600" model="account.account.template">
+        <field name="name">Биологические активы</field>
+        <field name="code">1600</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1700" model="account.account.template">
+        <field name="name">Прочие краткосрочные активы</field>
+        <field name="code">1700</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1010" model="account.account.template">
```

Comment:
it's already in the chart template as cash account prefix, so no need of this record

Reply:
I'm not sure. Kazakh accountants may want to use just `1010` account instead of `1010.1`